src/software/backend/input/network/networking/proto_multicast_listener.tpp

```diff
@@ -1,11 +1,10 @@
-#include "software/backend/input/network/networking/ssl_vision_client.h"
-
 #include "software/logger/logger.h"
```

Comment:
Since this is technically a header now, we should have a `#pragma once` at the top of this file to prevent circular includes (this has definitely tripped me up before).

Reply:
that's done